modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,141 @@
+
+With this update, {pipelines-title} General Availability (GA) 1.9 is available on {product-title} 4.10, and is planned to be available on {product-title} 4.11 and later.
+
+[id="breaking-changes-1-9_{context}"]
+== Breaking changes
+
+// .Pipelines
+
+* With this update, the `Conditions` custom resource definition (CRD) type has been removed. As an alternative, use the WhenExpressions CRD instead.
+
+* With this update, support for `tekton.dev/v1alpha1` API pipeline resources, such as, Pipeline, PipelineRun, Task, Clustertask, and TaskRun has been removed.
+
+* With this update, the `tkn-pac setup` command has been removed. You can now re-use the functionality provided by the `tkn-pac setup` command by using the `tkn-pac webhook add` and `tkn-pac webhook update-token` commands. You can use the `tkn-pac webhook add` command to re-add a webhook to an existing Git repository. And, you can use the `tkn-pac webhook update-token` command to update the personal provider access token for an existing Secret object in the Git repository.
+
+[id="deprecated-features-1-9_{context}"]
+== Deprecated and removed features
+
+* In the Red Hat OpenShift Pipelines 1.9 release, `ClusterTasks` are deprecated and planned to be removed in a future release. As an alternative, you can use `Cluster Resolver`.
+
+* In the Red Hat OpenShift Pipelines 1.9 release, the `Trigger` object and the `namespaceSelector` field in the `EventListener` specification are deprecated and planned to be removed in a future release.
+
+* In the Red Hat OpenShift Pipelines 1.9 release, the `tkn pipelinerun describe` command does not display timeouts for the `PipelineRun` resource.
+
+* In the Red Hat OpenShift Pipelines 1.9 release, the the `PipelineResource` custom resource (CR) is deprecated. The `PipelineResource` CR was a Tech Preview feature and part of the `tekton.dev/v1alpha1` API.
+
+* In the Red Hat OpenShift Pipelines 1.9 release, custom image parameters in cluster tasks are deprecated. You must run an operator-provided cluster task with a custom image. The following parameters must be configured in the custom image:
+
++
+[source,yaml]
+----
+buildah task doesn't expose "BUILDER_IMAGE" param
+git-clone task doesn't expose "gitInitImage" param
+kn task doesn't expose "kn-image" param
+kn-apply task doesn't expose "KN_IMAGE" param
+maven task doesn't expose "MAVEN_IMAGE" param
+s2i-* tasks don't expose "BUILDER_IMAGE" param
+tkn task doesn't expose "TKN_IMAGE" param
+----
+
+[id="known-issues-1-9_{context}"]
+== Known issues
+
+// .Operator
+
+* The `chains-secret` and `chains-config` config map are removed after you uninstall the Red Hat OpenShift Pipelines operator. As they contain user data, they should be preserved and not deleted.
+// https://issues.redhat.com/browse/SRVKP-2396
+
+// .PAC
+
+* When running the `tkn pac` set of commands on Windows, you may receive the following error message: Command finished with error: not supported by Windows.
```

Comment:
Hi @concaf - I have also updated the content with these set of known issues that you suggested some time ago, could you please take a look at these as well?

Reply:
looks good, thanks :)

Reply:
Thanks.